docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -1,14 +1,141 @@
 = Getting Started
 
-Kaskada's query language xref:overview:what-is-kaskada.adoc[focuses on event-based data processing] making it easier to reason about when things happen, the state at specific points in time, and how results change over time.
-Kaskada queries are simple, declarative and composable providing a more expressive way to query event-based data.
+Kaskada is a unified event processing engine that provides stateful stream processing in a high-level query language designed specifically for reasoning about events in bulk and in real time.
 
-There are several ways to quickly get started with Kaskada.
+This document will show you how to quickly get started using Kaskada.
 
-* xref:getting-started:hello-world-jupyter.adoc[]: How to get up and running with Kaskada inside a Jupyter notebook running locally. 
-This quick tutorial will help you install and run Kaskada from within a Jupyter notebook, load some sample data into Kaskada, use Kaskada queries and Python within Jupyter to explore that data.
-* xref:getting-started:hello-world-cli.adoc[]: How to execute queries from the command line using the Kaskada CLI. 
-This quick tutorial will help you install and run Kaskada from the command line, load some sample data into Kaskada, and use Kaskada queries to explore that data all on the command line.
+[TIP]
+====
+Before jumping in to writing queries in Kaskada, it's a good idea to take a minute understanding "how to think with Kaskada".
+Kaskada is built on simple, powerful abstrctions that may be different than what you're accustomed to.
```

Comment:
"simple, powerful abstractions":

1. Spelling
2. Suggest something like "Kaskada is built on timelines -- a simple, powerful abstraction that ..."

Specifically, "abstractions" makes it sound like there are several things to learn, while "timelines -- a simple, ..." makes it singular. May make it seem friendlier to learn just one thing.